src/vopr.zig

```diff
@@ -411,19 +407,15 @@ fn parse_args(allocator: mem.Allocator) !Flags {
     var args = try std.process.argsWithAllocator(allocator);
     defer args.deinit();
 
-    // Keep track of the args from the ArgIterator above that were allocated
-    // then free them all at the end of the scope.
+    // Keep track of the args from the ArgIterator above that were allocated.
+    // `args` will free them all at the end of the scope.
     var args_allocated = std.ArrayList([:0]const u8).init(allocator);
-    defer {
-        for (args_allocated.items) |arg| allocator.free(arg);
-        args_allocated.deinit();
-    }
+    defer args_allocated.deinit();
```

Comment:
Similarly to cli.zig, I _think_ that just `args` can now play the role we used `args_allocated` for before.